.github/workflows/create-release-branch.yml

```diff
@@ -0,0 +1,146 @@
+name: create-release-branch
+on:
+  workflow_dispatch:
+    inputs:
+      # When the workflow is executed manually, the user can select whether the branch should correspond to a major,
+      # minor, or patch release.
+      release-type:
+        type: choice
+        description: what kind of release?
+        options:
+          - major
+          - minor
+          - patch
+        required: true
+  schedule:
+    # Cron syntax is "minute[0-59] hour[0-23] date[1-31] month[1-12] day[0-6]". '*' is 'any value,' and multiple values
+    # can be specified with comma-separated lists. All times are UTC.
+    # So this expression means "run at 12 PM UTC, every Friday".
+    - cron: "0 12 * * 5"
```

Comment:
I debated having this on a delay relative to the scanner, but realized it would only matter if we were building our test tarball from the corresponding scanner release branch, which we currently aren't (see below). If we decide to change that, then I'll change this.